test/filter.jl

```diff
@@ -1,11 +1,11 @@
 using HDF5
+using HDF5.Filters
 using Test
+import HDF5.API: H5Z_FILTER_DEFLATE, H5Z_FILTER_SHUFFLE
```

Comment:
```suggestion
```

Reply:
Doesn't look like these are used

Reply:
Correct. It looks like their direct use was dropped in 2c85d18abd67988f0f09d10150b39db36c457257. I was just importing the constants rather than setting directly as before my change.